airbyte-integrations/connectors/source-google-pagespeed-insights/source_google_pagespeed_insights/spec.yaml

```diff
@@ -0,0 +1,26 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/google-pagespeed-insights
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Google PageSpeed Insights Spec
+  type: object
+  required:
+    - api_key
+    - url
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      title: API Key
+      description: >-
+        Google PageSpeed API Key. See <a
+        href="https://developers.google.com/speed/docs/insights/v5/get-started#APIKey">here</a>.
+        The toke is case sensitive.
+      airbyte_secret: true
+    url:
```

Comment:
Most websites are likely to want to monitor a list of urls instead of a single one, so I think we should turn this in an `urls` array property!

Reply:
How can we reflect multiple URLs in the yaml?
The API unfortunately only allows to fetch the report for one URL at a time.
With the python SDK I guess we could implement a simple loop over all the urls. How would one implement such a "loop" in the lowcode SDK? Would this somehow work with Stream Slices with a List stream slicer?

Reply:
Yes, the [ListStreamSlicer](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/stream-slicers/#list-stream-slicer) should be the way to go!